packages/app/src/Commands/MakePageCommand.php

```diff
@@ -115,10 +115,10 @@ public function handle(): int
                 ->append('.blade.php'),
         );
 
-        $files = array_merge(
-            [$path],
-            $resourcePage === 'custom' ? [$viewPath] : [],
-        );
+        $files = [
+            $path,
+            ...$resourcePage === 'custom' ? [$viewPath] : [],
```

Comment:
I think for better readability we should wrap this:

```suggestion
            ...($resourcePage === 'custom' ? [$viewPath] : []),
```